server/utils/routeUtils.ts

```diff
@@ -3,20 +3,63 @@
 import type { RequestHandler, Router } from 'express'
 
 import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { AuditEventSpec } from '../middleware/auditMiddleware'
+import auditMiddleware from '../middleware/auditMiddleware'
```

Comment:
The `auditMiddleware` can be imported from the index. Could the type be exported/imported from the index too?